src/components/ListItem.jsx

```diff
@@ -16,20 +16,22 @@ export function ListItem({
 	const todaysDate = Timestamp.now();
 	const isChecked = subtractDates(todaysDate, dateLastPurchased);
 
-	// if dateLastPurchased is true subtract it from dateNextPurchased, else subtract dateCreated from dateNextPurchased to get the estimated number of days till next purchase
+	// Calculate the previous estimate based on the last purchase date or creation date
 	const previousEstimate = Math.ceil(
-		(dateNextPurchased && dateLastPurchased
+		(dateNextPurchased instanceof Timestamp && dateLastPurchased
```

Comment:
Wondering the purpose of `instanceof Timestamp` added here and in other places vs how it was before?